src/components/ThemeRegistry/ThemeRegistry.js

```diff
@@ -1,22 +1,39 @@
-'use client';
+"use client";
 
 // React
-import * as React from 'react';
+import * as React from "react";
 
 // Material UI
-import { ThemeProvider } from '@mui/material/styles';
-import CssBaseline from '@mui/material/CssBaseline';
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import CssBaseline from "@mui/material/CssBaseline";
 
 // Local
-import NextAppDirEmotionCacheProvider from './EmotionCache';
-import theme from './theme';
+import NextAppDirEmotionCacheProvider from "./EmotionCache";
 
+const darkTheme = createTheme({
+    palette: {
+        mode: "dark",
+    },
+});
```

Comment:
What was the reasoning behind creating a new MUI them object for light and dark?

Reply:
in ThemeRegistry line 39 its used with a ternary to actually switch the themes